### PR TITLE
[doc] Revise wrt test suite developer/user guide

### PR DESCRIPTION
**Pull request by haoxli**

### CODE REVIEW

File: doc/Web_Runtime_Test_Suite_Developer_Guide.md [hunk 1]
```diff
@@ -2,310 +2,76 @@
 
 ## 1. Overview
```
Comment (Honry) [inline doc/Web_Runtime_Test_Suite_Developer_Guide.md]: The serial number is unnecessary.

File: doc/Web_Runtime_Test_Suite_Developer_Guide.md [hunk 2]
```diff
@@ -2,310 +2,76 @@
...
+        list=[[1,2],[3]]
+        input_pair = all_pairs( list )
+        for e, v in enumerate(input_pair):
+            print e,v
 
+        ....0 [1, 3]
+        ....1 [2, 3]
+       ```
 
 ## Appendix 1 `tests.full.xml` and `tests.xml`
```
Comment (Honry) [inline doc/Web_Runtime_Test_Suite_Developer_Guide.md]: This item has been introduced in ./Web_Test_Suite_Developer_Guide.md, pls. remove it.

File: doc/Web_Runtime_Test_Suite_User_Guide.md [hunk 3]
```diff
@@ -19,55 +19,73 @@ This document provides method to run WRT Crosswalk Test Suite on TIZEN. You can
...
+        $ apt-get install build-essential
+        $ apt-get install debhelper
 
-        $ sdb shell "rpm -ivh /home/TIZEN_USER/content/tct/crosswalk-<version\>.i686.rpm"
+- Download crosswalk
 
-        $ sdb shell "rpm -ivh /home/TIZEN_USER/content/tct/tizen-extensions-crosswalk-<version\>.i686.rpm"
+  For Android: https://download.01.org/crosswalk/releases/crosswalk/android/canary/&lt;version&gt;/crosswalk-&lt;version&gt;.zip
 
+  For Deepi  : https://download.01.org/crosswalk/releases/crosswalk/linux/deb/canary/&lt;version&gt;/crosswalk_&lt;version&gt;.deb
```
Comment (Honry) [inline doc/Web_Runtime_Test_Suite_User_Guide.md]: 's/Deepi/Deepin'

**Comment by Honry** [comment]
LGTM, @jennycao, pls. also help review it.

**Comment by zqzhang** [comment]
LGTM

**Pull request merged**